sql-schema/282.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX notifications_id_user_id ON notifications_attribs (notifications_id, user_id);
```

Comment:
Remove this file. We rely on laravel migrations now

Reply:
aye